Formula/bower.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class Bower < Formula
+  desc "Package manager for the web"
+  homepage "https://bower.io/"
+  url "https://registry.npmjs.org/bower/-/bower-1.8.0.tgz"
+  sha256 "df634fcbb57f8877914ceeff794c88bca0c8a262d9c9f12f08c37ab13599aaa6"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"bower", "install", "jquery"
```

Comment:
Thanks @woodruffw; that makes sense! I actually had this implemented in cglong@783647e; I'll reimplement this check and push to the PR soon 😄